src/irsForms/worksheets/ChildTaxCreditWorksheet.ts

```diff
@@ -0,0 +1,66 @@
+import F1040 from '../F1040'
+import { Dependent, FilingStatus } from '../../redux/data'
+import { computeField, sumFields } from '../util'
+import { CURRENT_YEAR, QualifyingDependents } from '../../data/federal'
+
+export default class ChildTaxCreditWorksheet {
+  f1040: F1040
+
+  constructor (f1040: F1040) {
+    this.f1040 = f1040
+  }
+
+  qualifiesChild = (d: Dependent): boolean =>
+    d.qualifyingInfo !== undefined && (CURRENT_YEAR - d.qualifyingInfo.birthYear < QualifyingDependents.childMaxAge)
```

Comment:
This is good. I thought it should be <= reading this line, but the instructions say under 19 at the end of 2020, so this accurately excludes a child who is 19. Now that I'm seeing it again, `maxAge` sounds inclusive to me. But a child with age 19 is older than 19 years and therefore should be excluded. Kind of depends on the reader. Maybe just a comment where maxAge is defined would help.